API/Repositories/Interfaces/IMatchesRepository.cs

```diff
@@ -14,15 +14,9 @@ public interface IMatchesRepository : IRepository<Match>
 	Task<Match?> GetFirstMatchNeedingApiProcessingAsync();
 	Task<Match?> GetFirstMatchNeedingAutoCheckAsync();
 	Task<IList<Match>> GetNeedApiProcessingAsync();
-	Task<IEnumerable<Match>> GetByMatchIdsAsync(IEnumerable<long> matchIds);
+	Task<Match?> GetNoTrackingAsync(int id);
+    Task<IEnumerable<Match>> GetByMatchIdsAsync(IEnumerable<long> matchIds);
```

Comment:
Rename to `GetAsync` (I've been trying to reduce the amount of overly verbose names in all the repositories).